src/app/components/Transactions/LogEvent.tsx

```diff
@@ -179,8 +175,11 @@ const DecodedLogEvent: FC<{
               <AccountLink
                 address={event.body.owner}
                 scope={scope}
-                plain={addressSwitchOption === AddressSwitchOption.ETH}
+                plain={addressSwitchOption !== AddressSwitchOption.Oasis}
```

Comment:
What's the point of this change?

Reply:
Just more accurate: "We have oasis address, so `plain` when addressSwitchOption isn't what we have"

instead of "We have oasis address, so `plain` when addressSwitchOption is ETH"